### PR TITLE
add dw as an option to data viewing ux

### PR DESCRIPTION
**Pull request by minsa110**

### CODE REVIEW

**Comment by jmew** [comment]
It might be good to add a screenshot of Data Wrangler as well to the doc

**Review by ntrogh — APPROVED** [review]
Some minor text edits. A screenshot of the data wrangler functionality could be added, but should not detract from the overall tutorial experience (keep size minimal).

**Comment by jmew** [comment]
Would be good to update the page here as well with the same information, since theres a section about Data Viewer

https://code.visualstudio.com/docs/datascience/jupyter-notebooks

File: docs/datascience/data-science-tutorial.md [hunk 1]
```diff
@@ -90,6 +90,8 @@ This tutorial uses the [Titanic dataset](https://hbiostat.org/data/repo/titanic.
 
    ![Data viewer and variable explorer](images/data-science-tutorial/dataviewer.png)
 
+   Alternatively, you can use the data viewing experience offered by other extensions like [Data Wrangler](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.datawrangler). The Data Wrangler extension offers a rich user interface to show insights about your data and helps you perform data profiling, quality checks, transformations, and more. Learn more about the [Data Wrangler extension in our docs](https://code.visualstudio.com/docs/datascience/data-wrangler).
```
Comment (ntrogh) [inline docs/datascience/data-science-tutorial.md:93]:
```suggestion
   Alternatively, you can use the data viewing experience offered by other extensions like [Data Wrangler](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.datawrangler). The Data Wrangler extension offers a rich user interface to show insights about your data and helps you perform data profiling, quality checks, transformations, and more. Learn more about the [Data Wrangler extension in our docs](/docs/datascience/data-wrangler.md).
```

**Review by ntrogh — CHANGES_REQUESTED** [review]
Can you use relative links when referencing other docs articles?

**Comment by minsa110** [comment]
Thanks for the url catch, @ntrogh! Fixed them :)

**Review by ntrogh — APPROVED** [review]

**Pull request merged**